<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>545</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_0">
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>331</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="button_load">
         <property name="text">
          <string>Load Files</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="button_play">
         <property name="text">
          <string>GO PLAY!!!</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="txt_chooseMode">
         <property name="text">
          <string>Choose Mode</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_1">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>701</width>
        <height>441</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="3">
        <widget class="QPushButton" name="button_load_file">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QLabel" name="txt_move_check">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Not load yet</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="txt_game">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>GameData File :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QLabel" name="txt_game_check">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Not load yet</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="text">
          <string>./GameData.txt</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="txt_text_case_check">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Not load yet</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="txt_pokemon">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Pokemon File :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="button_start">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>START!!</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="txt_move">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Move File :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="5">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>WARNING! If the files have been loaded,then you load others</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QPushButton" name="button_load_file_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="text">
          <string>./MoveLib.txt</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="txt_pokemon_check">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Not load yet</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="txt_text_case">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Text case File :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="button_load_file_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEdit">
         <property name="text">
          <string>./case.txt</string>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QPushButton" name="button_goback">
         <property name="text">
          <string>Go Back</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="button_load_file_2">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="text">
          <string>./PokemonLib.txt</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>IT WILL BE COVERED</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>240</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPokemon">
    <property name="title">
     <string>Pokemon</string>
    </property>
   </widget>
   <addaction name="menuPokemon"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
